mintlify/snippets/kyc-onboarding.mdx

```diff
@@ -0,0 +1,632 @@
+---
+title: "KYC/Onboarding Flow"
+description: "Complete customer onboarding and KYC verification process"
+---
+
+<Info>
+The KYC/onboarding flow allows you to onboard customers through a hosted KYC link or direct API calls. This snippet covers both approaches with references to the OpenAPI schema.
+</Info>
+
+## Platform Regulation Status
+
+Your platform's regulatory status determines the KYC/onboarding requirements and available flows. This is configured in your platform settings and affects which verification methods are available.
+
+### Regulated Platforms
+
+<Check>
+**Regulated platforms** have lighter KYC requirements since they handle compliance verification internally.
+</Check>
+
+Regulated financial institutions can:
+
+- **Direct API Onboarding**: Create customers directly via API calls with minimal verification
+- **Internal KYC/KYB**: Handle identity verification through your own compliance systems
+- **Reduced Documentation**: Only provide essential customer information required by the UMA provider
+- **Faster Onboarding**: Streamlined process for known, verified customers
+
+<Tip>
+Regulated platforms should use the direct API onboarding flow since you handle KYC/KYB verification internally according to your regulatory requirements.
+</Tip>
+
+### Unregulated Platforms
+
+<Warning>
+**Unregulated platforms** require full KYC verification through hosted flows.
+</Warning>
+
+Unregulated platforms must:
+
+- **Hosted KYC Flow**: Use the hosted KYC link for complete identity verification
+- **Full Documentation**: Provide comprehensive customer information including all required PII
+- **Provider Verification**: Rely on UMA provider verification systems for compliance
```

Comment:
I'd delete this line. I'm not sure what it means and I think "UMA provider" doesn't mean much in this context.